submissions/gorodiskiy/HTML-CSS-Popup/css/main.css

```diff
@@ -184,4 +181,6 @@ figure img {
 .logo {
     max-width: 150px;
     margin-bottom: 16px;
-}
\ No newline at end of file
+}
+
+/*  */
```

Comment:
Remove all content on line 186 and leave all rest.
This red sign([Image](http://prntscr.com/prmvhj)) means that you don't have an empty line at the end of file.